### ISSUE TITLE
<grep> doesn't show the grep string that matched when used on Issue Detail/Background

### ISSUE THREAD

**Issue by claconsay** [wagiro/BurpBounty#20]
Great extender! I just used it but it is now one of my favorite Burp Suite Extension. I have some few issues though.

I noticed that the other profiles are using `<grep>` tag on the Issue Details, I'm not really sure what is its function exactly, but I assumed that this should show the grep string that was found. But this doesn't worked as I expected, please feel free to correct me if I misunderstood it. 

Basically, what I want is to see on the issue detail which grep were found on the response. So, here's my profile setup:

**Scanner:** Passive Response
**Grep:** email, password, session
**Match type:** Regex
**Issue Detail:** The following string/s were found on the response: `<grep>`

I hope you'll be able to help me with this. I'm using v3.0.2beta

Thanks,
Chris

**Comment by wagiro** [comment]
Hi @claconsay! Thanks for your words. The <grep> tag for issue detail its for extract payload that generated the issue. You can see the string found in issue response highlighted. Sorry if the <grep> string its not the best word for this, maybe in next versions I will change. Regards!

**Comment by claconsay** [comment]
Hi @wagiro, thanks for the prompt reply. As a request, maybe you can also create a tag that will show the matched grep on issue details.

So that if the response contains a string "email" and we put this description on the Issue Details
`The following string/s were found on the response: <matched_grep>`

The output will look like
`The following string/s were found on the response: email`

Thanks.

**Comment by wagiro** [comment]
Hey @claconsay, it can be a good feature, <payloas> for extract payload and <grep> for extract a matched grep. I will add in future versions. Thanks!
